crates/bevy_ui/src/node_bundles.rs

```diff
@@ -278,6 +271,10 @@ impl Default for ButtonBundle {
             inherited_visibility: Default::default(),
             view_visibility: Default::default(),
             z_index: Default::default(),
+            pickable: Pickable {
+                should_block_lower: true,
+                ..Default::default()
+            },
```

Comment:
```suggestion
            pickable: Pickable::default(),
```
`should_block_lower` is `true` by default

Reply:
Great; that means we can swap to a pure derive then.